src/examples/rest/src/admin-ui/custom-fields/link.tsx

```diff
@@ -1,13 +1,20 @@
 import { MouseEventHandler } from 'react';
 
-import { Task } from '../../types.generated';
 import { ReactComponent as OpenIcon } from '../assets/16-open-external.svg';
+import { CustomFieldArgs } from '@exogee/graphweaver-admin-ui-components';
 
-export const Link = (task: Task) => {
+interface Task {
+	user: {
+		label: string;
+		value: string;
+	};
+}
+
+export const Link = ({ entity }: CustomFieldArgs<Task>) => {
 	const handleClick = (e: MouseEventHandler<HTMLDivElement>) => {
 		e.preventDefault();
 		e.stopPropagation();
-		window.open(`https://google.com/search?q=${task.user.name}`, '_blank', 'noreferrer');
+		window.open(`https://google.com/search?q=${entity.user.label}`, '_blank', 'noreferrer');
```

Comment:
Does the component get all the data on that row?

Reply:
Yup. `entity` is the row object.